integrations/gcp/gcp_core/utils.py

```diff
@@ -72,16 +77,48 @@ def get_current_resource_config() -> (
 
 
 def get_credentials_json() -> str:
-    b64_credentials = ocean.integration_config["encoded_adc_configuration"]
-    credentials_json = base64.b64decode(b64_credentials).decode("utf-8")
+    credentials_json = ""
+    if ocean.integration_config["encoded_adc_configuration"]:
```

Comment:
might be worth being on the safe side? 
```suggestion
    if ocean.integration_config.get("encoded_adc_configuration"):
```